.github/workflows/setup_miniconda.sh

```diff
@@ -1,7 +1,12 @@
-# This is a bash INCLUDE file. Source it don't run it.
+#!/bin/bash
```

Comment:
I'm not 100% sure we are assuming bash is available everywhere. Let's try to be Bourne Shell compatible if possible. It's useful on OSX because the default shell is `zsh`.